.github/CONTRIBUTING.md

```diff
@@ -10,15 +10,15 @@ There are many ways to get in touch:
 * request features
   * support for existing databases / APIs
   * integrating existing algorithms into the pipeline
-* join discussion of open issues, features, and bugs
-  * computing pipeline: github.com/terraref/computing-pipeline/issues
-  * reference data: github.com/terraref/reference-data/issues
+* join discussion of open issues, features, and bugs at AgPipeline/issues-and-projects/issues
 * contribute algorithms to the pipeline
 * revise documentation
 
-New features are directed toward developing and extending the TERRA Ref computational infrastructure. 
+New features are directed toward developing and extending the AgPipeline computational infrastructure. 
 
-If you need any help, please contact us in our [chat room](https://gitter.im/terraref/reference-data) or creating a [new issue](https://github.com/terraref/computing-pipeline/issues/new).
+If you need any help, please create a [new issue](https://github.com/terraref/computing-pipeline/issues/new).
```

Comment:
maybe give other options like emailing the maintainer? searching issues-and-templates issues to see if it has come up before?

Reply:
Good suggestion. Adding

Reply:
Changed to reflect this and also added more links to our OSF page on Issues and Pull Requests: https://osf.io/tzmhp/wiki/Issues%20and%20Pull%20Requests/